README.md

```diff
@@ -61,20 +86,56 @@ If those don't work and lead to an "Access denied" error please try the rules by
 - Change directory into the repository: `cd apex-tux`
 - Compile the app using the features you want
   - If you **don't** run DBus you have to disable the dbus feature: `cargo build --release --no-default-features --features crypto,usb`
-  - Otherwise just run `cargo build --release`
+  - Otherwise just run `cargo build --release --features sysinfo,hotkeys,gif`
```

Comment:
At least hotkeys should be included by default.

We could add a feature list in the **usage** section.

Reply:
I think the hotkeys do not work on windows because it's not inside of an event loop, so I don't know about that

but I think gif/image should be included by default since it doesn't have any OS requirement at all

Reply:
Ok, we should remove hotkeys too, or get any way to make it work on Windows.

Reply:
yup but as I said I can't get on windows rn, and there are other error before that, that are needed to be fixed